backend/src/main/java/nl/itvitae/buildachar/character/PlayerCharacterController.java

```diff
@@ -180,4 +181,60 @@ public ResponseEntity<PlayerCharacterGetDTO> getById(
         user.map(value -> value.getId().equals(character.getUser().getId())).orElse(false);
     return ResponseEntity.ok(PlayerCharacterGetDTO.from(character, isOwned));
   }
+
+  // van laila
+  @GetMapping("/search")
+  public ResponseEntity<Set<PlayerCharacterDetailsDTO>> getAll(
+      @RequestParam(required = false) Set<String> race,
+      @RequestParam(required = false, name = "class") Set<String> characterClass) {
```

Comment:
the two getAll endpoints should be merged into one (one endpoint with race, characterClass and ownedOnly request params

Reply:
i think the /find endpoint for findByNameContaining should also be added here as a 4th request param